### PR TITLE
Added company to list

### PR DESCRIPTION
**Pull request by Metaluim**

### CODE REVIEW

**Comment by marmelo** [comment]
Hi @Metaluim, can you please review the following items?

1. End the description sentence with a `.` period;
2. Switch order with OutSystems so that the list is sorted alphabetically;
2. Check if you do not want to provide a careers page.

Thanks!

**Comment by marmelo** [comment]
Hi @Metaluim, company categories have been added.
Can you please resolve conflicts and pick one?

Thanks!

**Comment by Metaluim** [comment]
Issues solved and conflict resolved.

**Comment by marmelo** [comment]
Hi @Metaluim, I believe the conflict resolution went wrong because some companies are duplicated.
Can you please check? Thanks, and sorry for the trouble!

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Kwanko](https://kwanko.com/) [:rocket:](https://www.kwanko.com/en/careers-join-kwanko-group-check-out-our-job-offers/#offres) | An International leader in Crossdevice Performance Marketing. | `Lisboa` |
 | [Mindera](https://mindera.com) [:rocket:](https://mindera.com/#we-are-hiring) | Technology development and consultancy. | `Aveiro` `Porto` |
 | [Moxy](https://moxy.studio) [:rocket:](https://moxy.studio/team#join-the-team) | Interdisciplinary studio focused on design and development. | `Porto` |
+| [OLX](https://www.olxgroup.com) [:rocket:](https://www.olxgroup.com/search/engineering/portugal-lisbon/all-brands) | Network of market-leading trading platforms across 5 countries. | `Lisboa` |
```
Comment (marmelo) [inline README.md]: These lines should not be added.

**Comment by vpereira01** [comment]
ping @Metaluim

**Comment by marmelo** [comment]
Thanks @Metaluim @vpereira01, I have made the changes to unblock this.

**Pull request merged**